lib/generators/shopify_app/scopes_storage/templates/home_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  include ShopifyApp::EmbeddedApp
+  include ShopifyApp::RequireKnownShop
+  include ShopifyApp::ScopesVerification
```

Comment:
Following the rule of threes: We should really look into aggregating this into a single concern, kind of like `ShopifyApp::Authenticated` in a future release.

This aggregate concern would be basically `ShopifyApp::Unauthenticated` or something.